js/views/modals/purchase/Purchase.js

```diff
@@ -348,24 +354,33 @@ export default class extends BaseModal {
             this.listenTo(this.payment, 'walletPaymentComplete',
               (pmtCompleteData => this.completePurchase(pmtCompleteData)));
             this.$('.js-pending').append(this.payment.render().el);
+            segmentation.id = data.orderId;
```

Comment:
Is it ok to track orderId's?

@SamPatt @morebrownies

Reply:
Not sure @rmisio, seems like a business decision to me.

Reply:
I am taking it out, we thought we could use it to connect state change events in orders so we could see which percentage of them made it through what steps, but even though countly has the data I haven't found a way to use it usefully.